packages/cst-dts-gen/src/generate.ts

```diff
@@ -56,7 +56,7 @@ function genNodeInterface(node: CstNodeTypeDefinition) {
 function genNodeChildrenType(node: CstNodeTypeDefinition) {
   const typeName = getNodeChildrenTypeName(node.name)
 
-  return `export type ${typeName} = {
+  return `export interface ${typeName} = {
```

Comment:
I think you need to get rid of the `=` sign as well

`export interface Foo { ... }` vs `export interface Foo = { ... }`

Reply:
yes! whoops, missed that